package.json

```
@@ -84,9 +84,9 @@
     "rollup-plugin-terser": "^7.0.2",
     "sirv-cli": "^1.0.12",
     "standard-version": "^9.3.0",
-    "svelte": "3.53.1",
+    "svelte": "^3.53.1 || ^4.0.5",
```

We can let users install a wider range of dependencies in `peerDependencies`, but it probably makes sense for us to test against a more specific version 
```suggestion
    "svelte": "^4.0.5",
```